index.html

```diff
@@ -1,47 +1,83 @@
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
-  <link rel="stylesheet"  type="text/css" href="application.css">
-  </head>
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
 <body>
   <h1 class="title-tech">Front-End Tools</h1>
```

Comment:
You shouldnt have more than one `h1` per scree, so I think you should use an `h2` here